applications/fishing-map/src/features/workspace/workspace.selectors.ts

```diff
@@ -42,6 +42,14 @@ export const selectFishingDatasets = createSelector([selectFishingDataviews], (d
   return dataviews.flatMap((dataview) => getDatasetsByDataview(dataview))
 })
 
+export const selectTemporalgridDataviews = createSelector(
```

Comment:
There is already a [selector to get this dataview](https://github.com/GlobalFishingWatch/frontend/blob/2b46157ea7b80f776ae7a700299e394a869988f5/applications/fishing-map/src/features/workspace/workspace.selectors.ts#L27), I agree this is a better name so we could remove the previous one and update the imports to use this one

Reply:
Ups, didn't see that, will fix